.github/workflows/gradle.yml

```diff
@@ -152,9 +146,6 @@ jobs:
           GIT_REVISION: ${{ github.sha }}
           CORE_ONLY: true
 
-      - name: Ensure no file change
```

Comment:
the point of the test_docker command is to run acceptance tests in docker. it shouldn't be duplicating build checks.